ModelTraining/awsAudioTS2Mp3DownloadandConvert.py

```diff
@@ -0,0 +1,197 @@
+# Get Started
+# pip install ffmpeg-python python-dateutil
+# AND
+# winget install Gyan.FFmpeg
+# you may need to add the FFmpeg path to system/user PATH variable
+# "C:\Users\User\AppData\Local\Microsoft\WinGet\Links\ffmpeg.exe"
+# or
+# "C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Gyan.FFmpeg_Microsoft.Winget.Source_8wekyb3d8bbwe\ffmpeg-6.0-full_build\bin\ffmpeg.exe"
+
+# USAGE
+# python .\awsAudioTS2Mp3DownloadandConvert.py --date '2020-09-11 22:14:00 PST' --node rpi_orcasound_lab
+# AWS Bucket defaults to: streaming-orcasound-net. Otherwise use --awsBucket streaming-orcasound-net to change bucket name
+
+# ffmpeg -i '.\live.m3u8' -c copy -bsf:a aac_adtstoasc demo.mp4
+
+import datetime, argparse, os, requests, ffmpeg, shutil
+from dateutil import parser
+from glob import glob
+
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+
+locations = [
+    'rpi_bush_point',
+    'rpi_mast_center',
+    'rpi_north_sjc',
+    'rpi_orcasound_lab',
+    'rpi_port_townsend',
+    'rpi_sunset_bay'
+]
+
+CLI=argparse.ArgumentParser()
+CLI.add_argument(
+    "--date",
+    nargs=1,
+    type=str,
+    default=None,
+    help="The data of the audio data of interest"
+)
+CLI.add_argument(
+    "--node",
+    nargs=1,
+    type=str,
+    default='rpi_orcasound_lab',
+    help="The name of the node where the hydrophone is located. Default: rpi_orcasound_lab"
+)
+CLI.add_argument(
+    "--awsBucket",
+    nargs=1,
+    type=str,
+    default='streaming-orcasound-net',
+    help="The name of the AWS audio data is stored. Default: streaming-orcasound-net"
+)
+args = CLI.parse_args()
+
+assert(args.node in locations)
+
+s3_client = boto3.client('s3', config=Config(signature_version=UNSIGNED, region_name='us-east-1'))
+
+# test_epoch = 1539203407
```

Comment:
Please clean up testing comments as appropriate

Reply:
I removed the test code